installer/terraform/scripts/configure-gitlab.py

```diff
@@ -0,0 +1,30 @@
+from gitlab_personalaccesstoken import generate_personal_access_token
+from terraform_external_data import terraform_external_data
+
+import gitlab
+
+
+@terraform_external_data
+def get_gitlab_group(query):
+    token = generate_personal_access_token('mytoken', query['passwd'], query['gitlab_ip'])
+
+    gl = gitlab.Gitlab('http://{0}'.format(query['gitlab_ip']), api_version=4, private_token=token)
+
+    group = gl.groups.create({'name': 'SLF', 'path': 'slf', 'description': 'Jazz framework, templates and services'})
```

Comment:
@bleggett Need this repo to be created along with SLF.

>  '{"name":"CAS","path":"cas", "description": "User created services repository"}')

Reply:
Oh so we need the group but not the variable, ok.